enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk/app-embed-sdk.tsx

```diff
@@ -0,0 +1,27 @@
+import { StrictMode } from "react";
```

Comment:
I'm not sure if the directory name is intended; we're mixing snake case and kebab case like crazy here. can we stick to just one convention if possible?

Reply:
@WiNloSt It was intentional -- this follows the convention of the other `app-` entry points (e.g. `app-embed.js`). I'll move this file to live with its entry point friends to address your other concern of EE licensing 👍🏻

Reply:
Fixed in https://github.com/metabase/metabase/pull/56915/commits/bd189b7862dbba16903cc67643712c086603b74c - now it lives in the correct directory and has the same convention (i.e. `app-embed.js`, `app-embed-sdk.tsx`)